src/mscorlib/src/System/Threading/Tasks/Task.cs

```diff
@@ -2339,21 +2309,6 @@ void IThreadPoolWorkItem.ExecuteWorkItem()
         }
 
         /// <summary>
-        /// The ThreadPool calls this if a ThreadAbortException is thrown while trying to execute this workitem.  This may occur
-        /// before Task would otherwise be able to observe it.  
-        /// </summary>
-        void IThreadPoolWorkItem.MarkAborted(ThreadAbortException tae)
```

Comment:
We have backed away from removing this earlier to make this easier to share with Mono: 
 https://github.com/dotnet/coreclr/pull/9234#issuecomment-276646511